### PR TITLE
Add Starname (IOV)

### PR DESCRIPTION
**Pull request by davepuchyr**
## Description

Add IOV. The iov-mainnet-2 chain is built on cosmos-sdk, just like Cosmoshub, BandChain, etc.

## Testing instructions

Test in the same manner as BandChain was tested.

## Types of changes

* New feature (non-breaking change which adds functionality)

## Checklist

- [ ] Prefix PR title with `[WIP]` if necessary.
- [x] Add tests to cover changes as needed.
- [x] Update documentation as needed.
- [x] I have read the [guidelines](https://developer.trustwallet.com/wallet-core/newblockchain#integration-criteria) for adding a new blockchain
- [ ] If there is a related Issue, mention it in the description (e.g. Fixes #<issue_number> ).

### CODE REVIEW

File: docs/coins.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ This list is generated from [./coins.json](../coins.json)
 | 178     | POA Network      | POA    | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/poa/info/logo.png" width="32" />          | <https://poa.network>         |
 | 194     | EOS              | EOS    | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/eos/info/logo.png" width="32" />          | <http://eos.io>               |
 | 195     | Tron             | TRX    | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/tron/info/logo.png" width="32" />         | <https://tron.network>        |
+| 234     | IOV              | IOV    | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/iov/info/logo.png" width="32" />          | <https://starname.me>         |
```
Comment (catenocrypt) [inline docs/coins.md:31]: The URL is 404.  Please add chain logo.png and info.json to assets repository.

File: tests/IOV/TWCoinTypeTests.cpp [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+// Generated one-time (codegen/bin/cointests)
+//
+
+#include "../interface/TWTestUtilities.h"
+#include <TrustWalletCore/TWCoinTypeConfiguration.h>
+#include <gtest/gtest.h>
+
+
+TEST(TWIOVCoinType, TWCoinType) {
+auto symbol = WRAPS(TWCoinTypeConfigurationGetSymbol(TWCoinTypeIOV));
```
Comment (catenocrypt) [inline tests/IOV/TWCoinTypeTests.cpp:17]: Proper fomatting (indenting) please.

**Review by catenocrypt — CHANGES_REQUESTED** [review]

**Comment by catenocrypt** [comment]
Looks good.
Please also create a PR for https://github.com/trustwallet/assets repo, to add https://github.com/trustwallet/assets/tree/master/blockchains/iov/info, with logo.png and info.json (look at iotex for example).

File: tests/IOV/TWCoinTypeTests.cpp (hunk 2 above)
Comment (davepuchyr) [reply tests/IOV/TWCoinTypeTests.cpp]: Done via https://github.com/trustwallet/wallet-core/pull/1123/commits/3248d526f21884323bcb11cf0957a6ff9dd675f5.

**Comment by catenocrypt** [comment]
Upon more careful review, it looks like Starname does not meet our blockchain inclusion criteria.  Closing, sorry.

https://developer.trustwallet.com/wallet-core/newblockchain

**Pull request closed without merging**